core/include/moveit/task_constructor/task.h

```diff
@@ -105,6 +105,13 @@ class Task : protected WrapperBase
 	/// remove function callback
 	void erase(TaskCallbackList::const_iterator which);
 
+	typedef std::function<void(const SolutionBase& t)> SolutionCallback;
```

Comment:
Small nit, because I know you're reading Effective Modern C++: Item 9. Prefer alias declarations to typedefs. :-)

Reply:
I thought we should keep it consistent in this class